src/opamParserTypes.mli

```diff
@@ -10,61 +10,124 @@
 (**************************************************************************)
 
 (** Defines the types for the opam format lexer and parser *)
+module Common: sig
 
-(** Relational operators *)
-type relop = [ `Eq  (** [=] *)
-             | `Neq (** [!=] *)
-             | `Geq (** [>=] *)
-             | `Gt  (** [>] *)
-             | `Leq (** [<=] *)
-             | `Lt  (** [<] *)
-             ]
+  (** Relational operators *)
+  type relop = [ `Eq  (** [=] *)
+               | `Neq (** [!=] *)
+               | `Geq (** [>=] *)
+               | `Gt  (** [>] *)
+               | `Leq (** [<=] *)
+               | `Lt  (** [<] *)
+               ]
 
-(** Logical operators *)
-type logop = [ `And (** [&] *) | `Or (** [|] *) ]
+  (** Logical operators *)
+  type logop = [ `And (** [&] *) | `Or (** [|] *) ]
 
-(** Prefix operators *)
-type pfxop = [ `Not (** [!] *) | `Defined (** [?] *) ]
+  (** Prefix operators *)
+  type pfxop = [ `Not (** [!] *) | `Defined (** [?] *) ]
 
-type file_name = string
+  type file_name = string
 
-(** Source file positions: [(filename, line, column)] *)
-type pos = file_name * int * int
+  (** Source file positions: [(filename, line, column)] *)
+  type pos = file_name * int * int
 
-(** Environment variable update operators *)
-type env_update_op = Eq       (** [=] *)
-                   | PlusEq   (** [+=] *)
-                   | EqPlus   (** [=+] *)
-                   | ColonEq  (** [:=] *)
-                   | EqColon  (** [=:] *)
-                   | EqPlusEq (** [=+=] *)
+  (** Environment variable update operators *)
+  type env_update_op = Eq       (** [=] *)
+                     | PlusEq   (** [+=] *)
+                     | EqPlus   (** [=+] *)
+                     | ColonEq  (** [:=] *)
+                     | EqColon  (** [=:] *)
+                     | EqPlusEq (** [=+=] *)
+end
+
+module FullPos : sig
+
+  (** Source file positions *)
+  type file_name = Common.file_name
+  type pos =
+    { filename: file_name;
+      start: int * int; (* line, column *)
+      stop: int * int; (* line, column *)
+    }
+  type 'a with_pos =
+    { pelem : 'a;
+      pos : pos
+    }
+
+  type relop_kind = Common.relop
+  and rel_op = relop_kind with_pos
```

Comment:
It's slightly strange having OpamParserTypes.relop and OpamParserTypes.FullPos.rel_op - it feels that it would be better that `open OpamParserTypes` and `open OpamParserTypes.FullPos` give as similar a signature as possible? What's wrong with:
```suggestion
  type relop_kind = relop
  type relop = relop_kind with_pos
```

Reply:
In fact, this renaming don't originate from the `FullPos` modules [introduction](https://github.com/ocaml/opam-file-format/pull/24/commits/39e8100b401e0f8e0c9a826b6b05d5d38a3ecf3b#diff-034c9d7984c00f23420753a0c3c0f77402e6a87f5d068a8bb6103b80fa9a1798R27). It is an opam problem, as an include done in [`OpamTypes`](https://github.com/ocaml/opam/pull/4298/files#diff-011f82839082bb0a4d19ca919973f41e80fe5a7c37b6c849ebdd80325e8d73aeR24) and the type relop used as is for `formula`s & co.
And after that, It's just a cosmetic consistency change for `log_op` and `pfx_op`

Reply:
Can it be fixed in `OpamTypes` instead?

Reply:
If it is fixed in opam, we need to change the name of `relop` type. When I made those changes, I had in mind minimal changes in opam's api, only the external value type changed.

Reply:
Or not include `OpamParserTypes`, and  create all needed aliases.

Reply:
Ok, added some `open` in opamp, to remove the include, and then no more naming problem!